eks-distro-base/Makefile

```diff
@@ -7,25 +7,27 @@ BASE_IMAGE?=public.ecr.aws/amazonlinux/amazonlinux:2
 IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
 IMAGE_NAME?=eks-distro/base
 # This tag is overwritten in the prow job to point to the commit hash
-IMAGE_TAG?=$(shell git rev-parse HEAD | cut -c 1-8)
+IMAGE_TAG?=$(shell date "+%F-%s")
 IMAGE?=$(IMAGE_REPO)/$(IMAGE_NAME):$(IMAGE_TAG)
 
 ifeq ($(DEVELOPMENT),true)
 	BASE_IMAGE=amazonlinux:2
 endif
 
 .PHONY: local-images
-local-images:
-	# Sleep to wait for buildkitd to start up
-	sleep 10
-	buildctl \
-		build \
-		--frontend dockerfile.v0 \
-		--opt platform=linux/amd64 \
-		--opt build-arg:BASE_IMAGE=$(BASE_IMAGE) \
-		--local dockerfile=./ \
-		--local context=. \
-		--output type=oci,oci-mediatypes=true,name=$(IMAGE),dest=/tmp/eks-distro-base.tar
+local-images: 
+	$(eval RETURN_VALUE="$(shell ./check_update.sh $(IMAGE_REPO) $(IMAGE_NAME) $(IMAGE_TAG) --dry-run)")
+	if [ $(RETURN_VALUE) = "Dry run" ]; then \
```

Comment:
Do we even need to check for dry run here? If we pass in --dry-run, won't it always print that?